endpoints/bookstore-grpc/README.md

```diff
@@ -55,8 +55,8 @@ gRPC libraries and tools are installed, run:
         --include_imports \
         --include_source_info \
         --proto_path=. \
-        --python_out=generated_pb2 \
-        --grpc_python_out=generated_pb2 \
+        --python_out=. \
```

Comment:
will this put the files in the same directory?

Reply:
Yes, this wasn't previously needed because the sample relied on an beta version of gRPC that combined the protobuf generated source and the grpc generated source.

With this update, the protoc command generates 2 files, one of which depends on the other, with a specific module path (hence the location of these file at the root).